.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "react-static/.babelrc",
+  "plugins": ["react-hot-loader/babel"]
```

Comment:
I don't think we need hot reloading. Can we remove this?

Reply:
I will try and remove it